.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
GENERAL: 
We really liked the simple design and colour choice, all the functions work correctly even after we tried to break stuff, we like the pop up for a required field. 

Cool that you actually served the client on localhost, we didn't do that we just used http-server. 

Cool that you used a bundler.

Like that you used a hashing system for security purposes I'm guessing.